homeassistant/components/screenlogic/services.yaml

```diff
@@ -1,9 +1,11 @@
 # ScreenLogic Services
 set_color_mode:
-  target:
```

Comment:
Can we keep this parameter but make it optional and deprecated and use the old logic to find the coordinator if the user passes this parameter? Raise a repair issue if this parameter is used. Mark the repair issue `is_fixable=True` and `is_persistent=True`. Search for examples of those parameters in the code to see how other integrations create repair issues for deprecated service calls.

Reply:
`config_entry` and `target` are accepted for the previously existing service call, but `target` is no longer exposed in the UI. An issue is created on the use of `target` in the service call. New services only accept `config_entry`.
Felt like this was a good approach. The change no longer breaks existing automation or scripts, but new users will only be exposed to the newer `config_entry` selector.